### DISCUSSION TITLE
PhysicsClientSharedMemory::processServerStatus() missing support for CMD_CALCULATE_INVERSE_KINEMATICS_COMPLETED

### DISCUSSION THREAD

**Discussion by matpalm** [bulletphysics/bullet3#3529] [General]
CMD_CALCULATE_INVERSE_KINEMATICS_COMPLETED causes PhysicsClientSharedMemory::processServerStatus() [to declare a warning](https://github.com/bulletphysics/bullet3/blob/master/examples/SharedMemory/PhysicsClientSharedMemory.cpp#L619)

> Warning: b3Error[third_party/bullet/examples/SharedMemory/PhysicsClientSharedMemory.cpp,620]:
> Warning: Unknown server status

would write a pull request but unsure what behaviour you want in this case (?)

**Comment by erwincoumans** [comment]
Thanks for the report. We should add a case 'CMD_CALCULATE_INVERSE_KINEMATICS_COMPLETED' break and CMD_CALCULATE_INVERSE_KINEMATICS_FAILED: with a warning, similar to INVERSE_DYNAMICS.
